tutorials/fluid_modeling.ipynb

```diff
@@ -0,0 +1,935 @@
+{
```

Comment:
<p>Here it should be clearer that the 'EoS approach' refers to the particular implementation in PorePy that wraps the cubic EoS and similar, and not the modeling choice of using an EoS.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1385/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>I am not sure what you mean, because it is made for the choice of using an EoS.</p><p> </p><p>Should I emphasize that in theory it can be used for any computations?</p>